arch/loongarch/slide_hash_lasx.c

```
@@ -42,8 +42,15 @@ Z_INTERNAL void slide_hash_lasx(deflate_state *s) {
     uint16_t wsize = (uint16_t)s->w_size;
     const __m256i ymm_wsize = __lasx_xvreplgr2vr_h((short)wsize);
 
-    slide_hash_chain(s->head, HASH_SIZE, ymm_wsize);
-    slide_hash_chain(s->prev, wsize, ymm_wsize);
+    slide_hash_chain_static(s->head, HASH_SIZE, ymm_wsize);
+    slide_hash_chain_static(s->prev, wsize, ymm_wsize);
+}
+
+Z_INTERNAL void slide_hash_head_lasx(deflate_state *s) {
```

Cppcheck (2.21.0)

[style] The function 'slide_hash_head_lasx' is never used.

(unusedFunction)